go/libkb/util_windows.go

```diff
@@ -96,3 +108,78 @@ func SafeWriteToFile(g SafeWriteLogger, t SafeWriter, mode os.FileMode) error {
 	}
 	return err
 }
+
+// helper for RemoteSettingsRepairman
+func moveNonChromiumFiles(g *GlobalContext, oldHome string, currentHome string) error {
+	g.Log.Info("RemoteSettingsRepairman moving from %s to %s", oldHome, currentHome)
+	files, _ := filepath.Glob(filepath.Join(oldHome, "*"))
+	for _, oldPathName := range files {
+		_, name := filepath.Split(oldPathName)
+		// Chromium seems stubborn about these - TBD
+		switch name {
+		case "GPUCache":
+			continue
+		case "lockfile":
+			continue
+		case "app-state.json":
+			continue
+		case "Cache":
+			continue
+		case "Cookies":
+			continue
+		case "Cookies-journal":
+			continue
+		case "Local Storage":
+			continue
+		case "started.txt":
+			continue
+		}
+		newPathName := filepath.Join(currentHome, name)
+		var err error
+		for i := 0; i < 5; i++ {
+			if err != nil {
+				time.Sleep(100 * time.Millisecond)
+			}
+			err = os.Rename(oldPathName, newPathName)
+			if err == nil {
+				break
+			}
+		}
+		if err != nil {
+			g.Log.Error("RemoteSettingsRepairman error moving %s to %s - %s", oldPathName, newPathName, err)
+			// Log files give problems, let's not fail on those
+			if ! strings.HasSuffix(name, ".log") {
```

Comment:
@maxtaco Even with the timeouts and retries, logs still fail, and this seems more reasonable than spending more hours trying to figure out why. This was added after your thumb, so I'm checking once again.